cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -1294,6 +1300,7 @@ impl TryFrom<api_types::PackageResolutionInfo> for SearchResult {
             version: Some(package_info.version),
             description: package_info.description,
             license: package_info.license,
+            pkg_path,
```

Comment:
note nonblocking: I wonder if we should cut out `SearchResult` from the `PackageInfoSearch` -> `SearchResult` -> `DisplayItem` chain. I think `SearchResult` is probably cruft leftover from pkgdb at this point. And we could probably use `PackageResolutionInfo` directly for show rather than `SearchResult` as well

Reply:
Besides exposing _generated_ api types directly without a shim seems a bit fragile, i agree `SearchResult` is/was a common intermediate of pkgdb and the catalog (and has ill named fields i.e. `rel_path`).

Trivia: `rel_path` makes sense from a flake view but is kinda redundant now that we mostly hide that with the catalog service.